Add rate limiting header docs

danihodovic opened the pull request:
## Deployment checklist
- [ ] The rate limiting headers need to be flagged to `on` in production by a change in chef.
- [x] Dani has been able to preview the docs locally which needs to be done to ensure they look OK.
- [x] We still need to decide on whether to send the `x-contentful-ratelimit-reset` header on all request or only 429s for the CMA and CPA.

neonichu commented on apiary/cpa.html:
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
This file shouldn't be checked in.

neonichu commented on apiary/cma.apib:
```diff
@@ -59,11 +59,21 @@ Contentful uses optimistic locking. When updating an existing resource its curre
-The rate limit is implemented through a standard mechanism of the HTTP protocol (returning response code 429).
+The rate limits are implemented through a standard mechanism of the HTTP protocol (returning response code 429).
 
-:[Common Resource Attribute](_partials/cma/resource-attributes.apib)
```
This line shouldn't have been removed, please re-add it after the rate limits section.

danihodovic commented:
> This file shouldn't be checked in.

Clumsy on my part, fixed in f9cf68d

> This line shouldn't have been removed, please re-add it after the rate limits section.
 
Fixed in b7b0f5b

danihodovic commented:
Can someone have a :eyes: at this?

neonichu commented:
LGTM :+1:

benben commented on apiary/cda.apib:
```diff
@@ -21,10 +21,17 @@ As an authorization HTTP request-header field and as an URI query parameter name
 Rate limits refer to the amount of requests per unit of time that can be done against our APIs.
-On the Content Delivery API there are no limits enforced on requests that hit the cache (i.e. a cache hit does not count towards rate limit and an unlimited amount of cache hits can be done). Nonetheless, there is still a rate limit of approximately 80 requests per seconds on the origin.
-
+On the Content Delivery API there are no limits enforced on requests that hit the cache (i.e. a cache hit does not count towards rate limit and an unlimited amount of cache hits can be done). Nonetheless, there is still a rate limit of approximately 21600 requests per hour and 80 requests per seconds on the origin.
```
why approximately? we have exact limits set for that.

danihodovic replied:
Because I didn't change the original docs :trollface:

benben replied:
please remove the approximately and use the numbers behind you on the wall :)

benben replied:
maybe we should also add that higher rate limits may apply due to enterprise contracts

danihodovic commented:
> please remove the approximately and use the numbers behind you on the wall :)
> maybe we should also add that higher rate limits may apply due to enterprise contracts

Fixed in 7b12d14
